appengine/metadata/standard/ci-setup.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Node 20 is already the default, so we can remove this ci-setup.json file altogether :)

Reply:
removed